uitools/common/src/OverviewMapController.cpp

```diff
@@ -136,6 +172,44 @@ namespace Esri::ArcGISRuntime::Toolkit {
     }
     else if (auto mapView = qobject_cast<MapViewToolkit*>(m_geoView))
     {
+      // create a lambda to handle setting up the inset map and viewpoint
+      auto setupInsetMapForMap = [this](MapViewToolkit* mapView)
+      {
+        // create the map
+        auto map = new Map(BasemapStyle::ArcGISTopographic, m_insetView);
+
+        // set the initial viewpoint (scale = main maps's scale * scaleFactor)
+        auto initialViewpoint = mapView->map()->initialViewpoint();
+        const Viewpoint newViewpoint{
+                                     geometry_cast<Point>(initialViewpoint.targetGeometry()),
+                                     initialViewpoint.targetScale() * scaleFactor(),
+                                     initialViewpoint.rotation()};
+
+        // set the initial viewpoint before setting on the mapview
+        map->setInitialViewpoint(newViewpoint);
+        m_insetView->setMap(map);
+      };
+
+      // if the map is already loaded, setup the inset map
+      if (mapView->map()->loadStatus() == LoadStatus::Loaded)
+      {
+        setupInsetMapForMap(mapView);
+      }
+      // set up the inset map once the map is done loading
+      else
+      {
+        connect(mapView->map(), &Map::doneLoading, this, [setupInsetMapForMap, mapView](Error e)
```

Comment:
Likely this will rarely/never happen but I suppose it's theoretically possible that the map/scene loads after you check `if (mapView->map()->loadStatus() == LoadStatus::Loaded)`, but before you connect to doneLoading. No great solutions to that that wouldn't involve a lot more code.

I think we should prevent a possible data race if possible, but this could scope creep your PR. What do you think?